charts/observability-pipelines-worker/values.yaml

```diff
@@ -33,6 +33,7 @@ datadog:
   ## Set to 'us3.datadoghq.com' to send data to the US3 site.
   ## Set to 'us5.datadoghq.com' to send data to the US5 site.
   ## Set to 'ddog-gov.com' to send data to the US1-FED site.
+  ## Set to 'ap1.datadoghq.com' to send data to the AP1 site.
```

Comment:
🤦 Ahhh sorry, I had thought I'd picked this change into #1082 but I missed it somehow. Feel free to bump the chart to version `1.2.2` to include this doc change.

Reply:
Ahhh @001wwang NP at all~ Got it! Thank you for supporting explanation. I'll bump the version and re-commit.

Reply:
I've done. https://github.com/DataDog/helm-charts/pull/1081/commits/5fbe8e633d0a429bc30afc2cc6f03819bd2333d9